lib/dor/text_extraction/abbyy.rb

```diff
@@ -2,77 +2,105 @@
 
 module Dor
   module TextExtraction
-    # Build Abbyy tickets for processing
     class Abbyy
-      attr_reader :filepaths, :druid
+      # Build Abbyy tickets for processing
+      class Ticket
+        attr_reader :filepaths, :druid
 
-      def initialize(filepaths:, druid:)
-        @filepaths = filepaths
-        @druid = druid
-      end
+        def initialize(filepaths:, druid:)
+          @filepaths = filepaths
+          @druid = druid
+        end
 
-      def write_xml_ticket
-        File.write(abbyy_ticket_filepath, xml_ticket)
-      end
+        def write_xml_ticket
+          File.write(abbyy_ticket_filepath, xml_ticket)
+        end
```

Comment:
could we simplify these method names now that this is called `Ticket`? like this could be `write_xml`, `abbyy_ticket_filepath` could just be `file_path`, etc. (which is a sign that this is a good change, imo!)

Reply:
That makes sense. Thanks. Done.